README.md

```diff
@@ -77,13 +77,13 @@ If Spark needs to be installed, that can be done using the following sparklyr co
 
 ``` r
 library(sparklyr)
-spark_install(version = "1.6.2")
+spark_install(version = "2.0.0")
 ```
 
-
-The call to `library(rsparkling)` will make the H2O functions available on the R search path and will also ensure that the dependencies required by the Sparkling Water package are included when we connect to Spark. 
-
+The call to `options(rsparkling.sparklingwater.version = ...)` will globally set up a specific Sparkling Water version. This is the version of Sparkling Water that will be called in the `library(rsparkling)` command, which will make the H2O functions available on the R search path and will also ensure that the dependencies required by the Sparkling Water package are included when we connect to Spark. 
```

Comment:
`library(rsparkling)` will not attach h2o functions to search path. This changed a while ago when h2o was moved from Depends to Imports. To use `h2o` functions user now need to load h2o package separately with `library(h2o)`.